server/routes/accounts.js

```diff
@@ -25,13 +26,16 @@ const getBankAccounts = (req, res) => {
 }
 
 const signBankAccount = (req, res) => {
-  const { ethAccount, accessToken, accountId } = req.body
+  const { ethAccount, token, accountId } = req.body
```

Comment:
I had to change this because the frontend doesn't have the `accessToken` to send here.